hedyorg/hedy
# 📜 rename the teacher manual section in the yaml files #4701

Fixes issue #4701 

**How to test**

Follow these steps to verify this PR works as intended:

I have changed the KEY "section" to teachers-manual" in all. YAML files, which are in content/pages/.yaml.
This should just provide a better naming for the KEY. The change is internal, and it should not affect the website.
To verify everything is working, navigate with a teacher account to the teacher manual. If there is text, it all works.


**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done.)


If you're unsure about any of these, don't hesitate to ask. We're here to help!
